pango/sys/Cargo.toml

```
@@ -54,3 +55,6 @@ version = "1.46"
 
 [package.metadata.system-deps.pango.v1_48]
 version = "1.48"
+
+[package.metadata.system-deps.pango.v1_50]
+version = "1.50"
```

You probably want something like

```suggestion
version = "1.49.2"
```
here or not? There's no 1.50 yet